test/kubetest/kubetest.go

```diff
@@ -0,0 +1,154 @@
+package kubetest
+
+import (
+	"testing"
+	"time"
+
+	"k8s.io/apimachinery/pkg/util/rand"
+	"k8s.io/client-go/kubernetes"
+	"k8s.io/client-go/tools/clientcmd"
+)
+
+type Suite struct {
```

Comment:
Can you add some doc comments to these structs? What is a `Suite`, what can it evolve into? What is it not?

Reply:
I am currently thinking about moving this package into its own repository for quicker experiments and proper CI with linting etc. :+1: